liferay-echo/liferay-portal
# LPS-195935 Add infopanel for LayoutPageTemplateEntries and LayoutPageTemplateCollections

[Story](https://liferay.atlassian.net/browse/LPS-195935)

Motivation
=======
The infopanel is currently now available for folders or DPTs created in Design > Page Templates > Display Page Templates.

Proposed Solution
========
Add the infopanel.

Steps to Verify
========
1. Go to Design > Page Templates > Display Page Templates,  create several folders (nested ones) and DPTs.
2. Ensure the infopanel show the right information in the following way:
3. When nothing is selected, the infopanel show the current folder info
4. When a DPT is selected, the infopanel shows the information as described in the [figma](https://www.figma.com/file/ydHldCxSGZuPe1tuHu9jXe/LPS-189856---Usability-improvements-for-objects-page-builder-integration?type=design&node-id=1-453&mode=design&t=dcDO8RIFKzjTGYaU-0) attached to the original story.
5.  When a folder is selected, the infopanel shows the information as described in the [figma](https://www.figma.com/file/ydHldCxSGZuPe1tuHu9jXe/LPS-189856---Usability-improvements-for-objects-page-builder-integration?type=design&node-id=1-453&mode=design&t=dcDO8RIFKzjTGYaU-0) attached to the original story.
6. When multiple items are selected, the info panel inly show the number of elements that have been selected.

Commits
========

- LPS-195935 Create webkeys to allow selection of LayoutPageTemplatesEntries and LayoutPageTemplatesCollections to apply different actions
- LPS-195935 Add method to retrieve actual LayoutPageTemplateCollection
- LPS-195935 Override getInfoPanelId method
- LPS-195935 Override getInputName methods to know selected LayoutPageTemplateEntries and LayoutPageTemplateCollections
- LPS-195935 Add infopanel option to LayoutPageTemplates view
- LPS-195935 Display infopanel for LayoutPageDisplayCollections
- LPS-195935 Showing current layoutPageTemplateCollection info in infopanel when nothing is selected
- LPS-195935 Delete layoutPageTemplateCollection and layoutPageTemplateEntries from Display Page Templates tab
- LPS-195935 Display infopanel for LayoutPageTemplateEntries
